geospaas_rest_api/tests.py

```diff
@@ -1,3 +1,76 @@
 from django.test import TestCase
+from django.conf import settings
+from django.test import Client
+from geospaas.vocabularies.models import Platform, Instrument, Parameter
+from geospaas.vocabularies.models import ISOTopicCategory, DataCenter
+from geospaas.catalog.models import *
+import json
 
 # Create your tests here.
+class DatasetURITests2(TestCase):
+
+    fixtures = ["vocabularies", "catalog"]
+
+    def test_api_initial_calls(self):
+
+        c = Client()#self.dataset = Dataset.objects.get(pk=1)
+        
+        response = c.get('//api/')
+        self.assertEqual(response.status_code, 200)
+
+        response2 = c.get('//api/geographic_locations/')   
+        self.assertEqual(response2.status_code, 200)
+        self.assertIn(b'geometry', response2.content)
+        
+        
+        response3 = c.get('//api/sources/')
+        self.assertEqual(response3.status_code, 200)
+        self.assertIn(b'specs', response3.content)
+ 
```

Comment:
Please red PEP8 coding style and try to write in accordance:
https://www.python.org/dev/peps/pep-0008/#blank-lines
### PEP8:
```
Blank Lines
Surround top-level function and class definitions with two blank lines.

Method definitions inside a class are surrounded by a single blank line.

Extra blank lines may be used (sparingly) to separate groups of related functions. Blank lines may be omitted between a bunch of related one-liners (e.g. a set of dummy implementations).

Use blank lines in functions, sparingly, to indicate logical sections.

Python accepts the control-L (i.e. ^L) form feed character as whitespace; Many tools treat these characters as page separators, so you may use them to separate pages of related sections of your file. Note, some editors and web-based code viewers may not recognize control-L as a form feed and will show another glyph in its place.
```

Reply:
With a few VS Code extensions this can pretty much be done automatically